README.md

```diff
@@ -157,27 +110,19 @@ $ ansible-playbook -i localhost, --connection=local \
         kubevirt_mf=kubevirt.yaml" \
         install-kubevirt-on-openshift.yml
 ```
+where
+* `kubevirt_mf` is a path to KubeVirt manifest. See [releases](https://github.com/kubevirt/kubevirt/releases) or build it from sources,
+* `openshift_ansible_dir` is a path to a cloned [OpenShift Ansible][openshift-ansible-project] git repository,
+* `kconfig` is a path to the [`~/.kube/config`](https://docs.openshift.com/container-platform/3.7/cli_reference/manage_cli_profiles.html#switching-between-cli-profiles) file containing authentication information for the desired cluster.
```

Comment:
`kconfig` was changed to `kubeconfig` in https://github.com/kubevirt/kubevirt-ansible/pull/72

Reply:
@gbenhaim right, updated, thanks